### ISSUE TITLE
Cant force name via get5_addplayer

### ISSUE THREAD

**Issue by JesusWithPants** [splewis/get5#366]
Edit: Re-did as requested by @MikkelDK (as said - didnt knew if its a bug or I need more functioning neurons).
### Expected behavior
Upon executing `get5_addplayer [steamid] [team] "namegoeshere"` to add player with name "namegoeshere".

### Actual behavior
Player is added but set name isn't forced to player.

### Steps to reproduce
- Plugin version:0.7.1
- Sourcemod version:1.9
- Steps to reproduce (please be specific):
 1) Execute `get5_addplayer [steamid] [team] "namegoeshere"`
 2) See that "namegoeshere" is not forced upon player
 3) Profit?

Original post:
Cant tell if I need proper sanity check or its an actual bug:

when executing command get5_addplayer it shows:
`Usage: get5_addplayer <auth> <team1|team2|spec|> [name]`
So in theory You should be able to execute a command:
`get5_addplayer xxxxx team1 "namegoeshere"` (doesnt matter what format I enter the command). It executes, adds player, but doesnt force the name. Hence the question: how should the [name] part be placed into command, or is this a bug and something is broken?

P.S Also consider adding a `get5_listplayers` command to output each teams player steamID and name for convinience sake.

**Comment by MikkelDK** [comment]
The issue template is there for a reason, please do NOT delete it.
```
If making a feature request, you should delete all the pre-filled text here.
If reporting a bug, fill in the following sections:

### Expected behavior



### Actual behavior



### Steps to reproduce

Please note that "latest" is NOT a version! See https://github.com/splewis/get5/wiki/Debugging-help#finding-get5-version if you need help finding these. Adding the output file produced by ``get5_debuginfo`` is extremely helpful.

- Plugin version:
- Sourcemod version:
- Steps to reproduce (please be specific):
```

**Comment by JesusWithPants** [comment]
Edited to propper format, apologies!

**Comment by PhlexPlexico** [comment]
So do you run this before you start a match? Do the names update on match change? Scouring through some of the code, it looks like `g_PlayerNames` eventually gets set, going through a few functions and forwards, and then is called during `LoadPlayerNames` [here](https://github.com/splewis/get5/blob/30b5406c6e3ffac7313714729b31ae72ab993683/scripting/get5/matchconfig.sp#L133). 

May I ask the process you're using? Are you setting the names and teams first, then creating a match? Or are you creating a match, then adding people to teams? I'm not even sure if the first one is really possible, but yeah, it does properly set the names for each player and adds them to the config, you just need to update and reload the config by the looks of it?

**Comment by JesusWithPants** [comment]
A json via get5_loadmatch_url gets launched where the steamIDs and names per team are set, that works and forces names. But sometimes after match is loaded and in Warmup state a player needs added - and here lies the problem. I can add player, but at this state (where either game is in warmup,live or whatever) player is added via command, but name is not forced upon the player manualy added.

**Comment by splewis** [comment]
The root issue is that `LoadPlayerNames` is only called on match loads, so player names added later with get5_addplayer won't have any effect.

It might be possible to just call `LoadPlayerNames` again every time `g_PlayerNames` is changed, but I'm not sure if there's potentially any harmful side effects from the server running the `sv_load_forced_client_names_file` command repeatedly. I would expect it to be fine, though.

**Comment by JesusWithPants** [comment]
So Im not crazy and its a bug/limitation. \o/

Any ETA for a potential fix to try out?

**Comment by PhlexPlexico** [comment]
I made a small one-line change in [teamlogic.sp](https://github.com/splewis/get5/blob/master/scripting/get5/teamlogic.sp#L350) here. I can send you a compiled get5 and create a PR if this is an acceptable change here. I don't think it should affect anything performance wise, as this only gets re-called when a new player is added? I've only tested it with one player on the server, but it seems to be working for the small amount of testing I've done.
